.travis.yml

```diff
@@ -23,4 +23,4 @@ jobs:
       language: go
       go: 1.9
       script:
-      - make verify test
+      - make verify verify_pr test
```

Comment:
Didn't realise this file was still around - we don't use travis anymore. Thanks for the reminder to remove the file 😄

Reply:
I did notice the repo didn't use it; I wasn't sure if you wanted to keep it up to date for turning on later or something though.